File: blog-post-2/kotlin/ArmStrong.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var scanner = Scanner(System.`in`)
    print("Enter number: ")
    var number = scanner.nextInt()
    var temp = number
    var count = 0
    var result = 0
    while(number > 0 ){
        count++
        number = number/10
    }

    number = temp
    var reminder:Int
    while(number > 0){
        reminder = number % 10
        result = result.plus(Math.pow(reminder.toDouble(), count.toDouble())).toInt()
        number = number.div(10)
    }
    if(temp == result){
        println("Armgstrong number")
    }
    else{
        println("Not armgstrong number")
    }
}

File: blog-post-2/python/ArmStrong.py
from math import pow

number = int(input("Enter number: "))
temp = number
count = 0
result = 0
while number > 0:
    count += 1
    number = int(number/10)

number = temp
reminder = 0
while(number > 0):
    reminder = (number % 10)
    result += int(pow(reminder, count))
    number = int(number / 10)

if(temp == result):
    print("Armgstrong number")
else:
    print("Not armgstrong number")
